my-app/src/pages/Homepage/Homepage.css

```diff
@@ -0,0 +1,60 @@
+:root{
+  --shadow-color : black;
+  --hero-text-color : white;
+}
+
+.container_home {
+    display: grid;
+    column-gap:20px;
+    grid-template-columns: repeat(5, 1fr);
+    grid-template-areas:
+     
+      "head    head    head     head    head"
+      "section section section section  section"
```

Comment:
![image](https://user-images.githubusercontent.com/75421090/161411909-67c3bc3c-0ead-4b2a-bcf5-3e77b5f31712.png)
This space left is pointless, could do better layouting , as a personal preference I am not a fan of this structure.

Reply:
i Was doing with grid now will change to flex